shm/Cargo.cpp

```diff
@@ -0,0 +1,34 @@
+#include "Cargo.h"
+
+	Cargo::Cargo (std::string name, size_t amount, size_t basePrice) :
```

Comment:
Good idea will be to add .clang-format file, so all co-owners (workers ;p) will use the same formatting.

Reply:
you are right, its a good idea :D